easybuild/easyblocks/m/mvapich2.py

```diff
@@ -106,6 +97,11 @@ def sanity_check_step(self):
         Custom sanity check for MVAPICH2
         """
         custom_paths = {
-            'files': ['bin/%s' % x for x in ['mpiexec.mpirun_rsh']] 
+            'files': ['bin/mpiexec.mpirun_rsh'],
         }
-        super(EB_MVAPICH2, self).sanity_check_step(custom_paths=custom_paths)
+
+        # cfr. http://git.mpich.org/mpich.git/blob_plain/v3.1.1:/CHANGES
+        # MVAPICH2 2.1 is based on MPICH 3.1.4
+        use_new_libnames = LooseVersion(self.version) >= LooseVersion('2.1')
+
+        super(EB_MVAPICH2, self).sanity_check_step(custom_paths=custom_paths, use_new_libnames=use_new_libnames)
```

Comment:
if you call `sanity_check_steps`, it'll actually check the paths (and raise an error if they're not found)

however, you can achieve what you want, by making the `EB_MPICH` easyblock pick up `custom_paths['files']`

it doesn't do that yet, but I'll fix that